src/server/handler.rs

```diff
@@ -159,7 +155,23 @@ pub trait Handler<C: Connection> {
     ///The return value shall be either `None` (if `name` is not a valid
     ///property or the required modules were not yet agreed to), or the return
     ///value from `MessageFormatter::publish_property()`.
-    fn handle_property<'c>(&self, name: &str, requested_value: Option<&[u8]>, conn: &mut C, send_buffer: &mut [u8]) -> Option<usize>;
+    fn handle_sub<'c>(&self, name: &str, conn: &mut C, send_buffer: &mut [u8]) -> Option<usize>;
```

Comment:
These function names don't reflect what they're about, and `sub` is ambiguous. I suggest:

- `handle_sub` -> `subscribe_to_property`
- `handle_set` -> `set_property`